tests/security/ebpf/disable_unprivileged_ebpf.pm

```diff
@@ -0,0 +1,72 @@
+# Copyright 2021 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+#
+# Summary: Test 'Unprivileged eBPF usage has been disabled,
+#          the setting can be changed by the `root` user':
+#          '# Verify the eBPF should be disabled unprivileged eBPF by default'
+#          '# Verify re-enable unprivileged eBPF temporarily using "systemctl"'
+#          '# Verify re-enable unprivileged eBPF temporarily using status file'
+#          '# Verify re-enable unprivileged eBPF persistently using config file'
+# Maintainer: llzhao <llzhao@suse.com>
+# Tags: poo#103932, tc#1769831
+
+use base 'opensusebasetest';
+use power_action_utils "power_action";
+use strict;
+use warnings;
+use testapi;
+use utils;
+use Utils::Backends 'is_pvm';
+
+sub reboot_and_check {
+    my ($self, $status) = @_;
+    my $f_unpriv_bpf_disabled = '/proc/sys/kernel/unprivileged_bpf_disabled';
+
+    # Reboot and verify the eBPF status
+    power_action("reboot", textmode => 1);
+    reconnect_mgmt_console if is_pvm;
+    $self->wait_boot(textmode => 1, ready_time => 600, bootloader_time => 300);
+    select_console 'root-console';
+    validate_script_output("cat $f_unpriv_bpf_disabled", sub { m/$status/ });
+}
+
+sub run {
+    my ($self) = @_;
+    my $f_unpriv_bpf_disabled = '/proc/sys/kernel/unprivileged_bpf_disabled';
+    my $sysctl_conf = '/etc/sysctl.conf';
+
+    select_console 'root-console';
+
+    # Verify the eBPF status: OS should be disabled unprivileged eBPF by default
+    validate_script_output("cat $f_unpriv_bpf_disabled", sub { m/2/ });
+
+    # Re-enable unprivileged eBPF temporarily using 'systemctl'
+    validate_script_output('sysctl kernel.unprivileged_bpf_disabled=0', sub { m/kernel.unprivileged_bpf_disabled = 0/ });
```

Comment:
Lili,
One minor comment, do you have some test coverage using 
"sysctl -w" here and reboot to check, then it should be enabled?

Reply:
I use "# sysctl key=value" to change the key **temporarily** and did reboot also.
At the end I use another way to change the key **persistently** and did reboot also.
I think the coverage is OK since there are lots of ways to do these testings.
Those methods are offered by jira comments. I think they have no problem.